topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1214 @@
+>
+>     ```text
+>     SampleID	URL
+>     Short read sample 0	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_0_reads.tar.gz
+>     Short read sample 1	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_1_reads.tar.gz
+>     Short read sample 2	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_2_reads.tar.gz
+>     Short read sample 3	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_3_reads.tar.gz
+>     Short read sample 4	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_4_reads.tar.gz
+>     Short read sample 5	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_5_reads.tar.gz
+>     Short read sample 6	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_6_reads.tar.gz
+>     Short read sample 7	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_7_reads.tar.gz
+>     Short read sample 8	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_8_reads.tar.gz
+>     Short read sample 9	https://frl.publisso.de/data/frl:6425521/marine/short_read/marmgCAMI2_sample_9_reads.tar.gz
+>     ```
+>
+> 7. Click on the **Build** button
+> 8. Remove the header by
+>    1. Click on the **Filter** menu
+>    2. Select `First or Last N Rows`
+>    3. Select
+>       - *"Filter which rows?"*: `first`
+>       - *"Filter how many rows?"*: `1`
+>    4. Click `Apply`
+>
+> 9. Define the Identifier and URL columns by
+>    1. Click on the **Rules** menu
+>    2. Select `Add / Modify Column Definitions`
+>    3. Define the column with Identifier
+>       1. Click `Add Definition` button
+>       2. Select `List Identifier(s)`
+>       3. Select `A` column
+>    4. Define the column with URL
+>       1. Click `Add Definition` button
+>       2. Select `URL`
+>       3. Select `B` column
+>    5. Click on `Apply`
+> 10. Add the name `Short reads` to the collection
+> 11. Click on `Upload`
+>
+>     > <comment-title></comment-title>
+>     > The process will take time.
+>     {: .comment}
+>
+> 12. Inspect the generated collection
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. What is the format of the file for each sample?
+> 2.
+>
+> > <solution-title></solution-title>
+> >
+> > 1.
+> >
+> {: .solution}
+>
+{: .question}
+
+We got the marine data for short-reads in our history. Let's do the same for long-reads.
+
+> <hands-on-title>Upload long-read data into Galaxy with Rule Builder</hands-on-title>
+>
+> 1. Click the upload icon toward the top left corner.
+> 2. Click on `Rule-based` on the top right
+> 3. Select `Collection(s)` in **Upload data as**
+> 4. Paste the following tabular data right into the textbox:
+>
+>     ```text
+>     SampleID	URL
+>     Long read sample 0	https://frl.publisso.de/data/frl:6425521/marine/long_read/marmgCAMI2_sample_0_reads.tar.gz
```

Comment:
These links do not work: it is tar.gz with subfolders in. We will put the data in the shared data library